### PR TITLE
Added: Software - Flutter Bond

### PR DESCRIPTION
**Pull request by mohamedalwhaidi**
### What is your contribution type?

- [x] Suggest an Awesome item.
- [ ] Web site maintenance.
- [ ] Other.

### Checklist

- [x] Ensure to check our [Contribution Criteria](../blob/main/CONTRIBUTING.md#criteria).
- [x] The contribution doesn't already exist (you can use [search functionality](https://awesome-arabic-speakers.dev/search/)).
- [x] The contribution is provided in both Arabic and English.

### CODE REVIEW

File: content/post/projects/flutter-bond/index.ar.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+* إدارة حالة التطبيق بسلاسة.
+* إجراء طلبات API مبسطة باستخدام Bond Fire.
+* التنقل والتوجيه السهل بين الشاشات.
+* بنية قابلة للتوسيع (Bond Core).
+* دعم واجهة سطر الأوامر (Bond CLI) لتطوير أسرع.
+
+من خلال التركيز على تقليل التعليمات البرمجية المكررة، يهدف Flutter Bond إلى جعل تطوير تطبيقات الهاتف المحمول أكثر متعة وكفاءة.
+
+**تعلم فلاتر بوند**
```
Comment (aabouzaid) [inline content/post/projects/flutter-bond/index.ar.md]: To keep the website simple and avoid stale posts, we avoid adding details like this and leave it for the user to discover via the project URL.

So you can remove it from this line and afterward.

File: content/post/projects/flutter-bond/index.ar.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (aabouzaid) [inline content/post/projects/flutter-bond/index.ar.md]: That should be in the `software` folder.

File: content/post/projects/flutter-bond/index.ar.md (hunk 1 above)
Comment (mohamedalwhaidi) [reply content/post/projects/flutter-bond/index.ar.md]: Ok, I will do that

File: content/post/projects/flutter-bond/index.ar.md (hunk 2 above)
Comment (mohamedalwhaidi) [reply content/post/projects/flutter-bond/index.ar.md]: Should I move the flutter-bond folder to the software folder?

Comment (aabouzaid) [reply content/post/projects/flutter-bond/index.ar.md]: Yes, please.

**Review by aabouzaid — APPROVED** [review]
@mohamedalwhaidi Thanks for your contribution :raised_hands: 
I made a tiny tidy-up, so it's ready to merge.

**Pull request merged**